lib/cache.js

```diff
@@ -48,12 +50,8 @@ Cache.prototype.clr = function(prefix, fn) {
   });
 };
 
-// Extend Map interface with:
-// `cache.allocated` - total allocated size
-// `cache.add(key, val)` - add record
-// `cache.del(key)` - delete record
-// `cache.clr(prefix, fn)` - delete all if `key.startsWith(prefix)`
-// Returns: object, cache instance
+// Get Cache, enhanced Map
```

Comment:
s/Get Cache, enhanced Map/Create new Cache instance/

Reply:
There are similar comments in `events` https://github.com/metarhia/common/blob/comments-refactor/lib/events.js#L68 Should they be also changed?

Reply:
Yeah, but be sure to add the description to the `EnhancedEmmiter` class.